Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -235,3 +235,24 @@
   - type: Clothing
     sprite: Clothing/Eyes/Glasses/ninjavisor.rsi
   - type: FlashImmunity
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons]
```

Comment:
You want
[ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons]

I would also say remove the Construction component, you don't want the crew to craft these